_includes/language-support.html

```diff
@@ -51,8 +51,10 @@
         <tr>
             <td>Launch-with Platforms</td>
             <td class="danger">N/A</td>
-            <td class="success">FireCloud
-                <br>DNAStack (workflows only)</td>
+            <td class="success">
+                FireCloud (HTTP imports supported, but not file-based imports)
```

Comment:
is it workflows only for Firecloud as well? 

Please elaborate. Use full sentences and a legend as appropriate.

Reply:
@coverbeck to answer

Reply:
probably workflows only too

Reply:
We only added the button to workflows, so there's now way to export a tool to FC, and it doesn't look there is a way in the FC UI to select a tool. So, yes workflows only for FC.